cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.api.curatoroperator.io/customresourcedefinition.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  name: reportapis.curator.operatefirst.io
+spec:
+  group: curator.operatefirst.io
+  names:
+    kind: ReportAPI
+    listKind: ReportAPIList
+    plural: reportapis
+    singular: reportapi
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: ReportAPI is the Schema for the reportapis API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: ReportAPISpec defines the desired state of ReportAPI
+            properties:
+              metricsName:
+                description: Show report aforementioned metrics only
+                items:
+                  enum:
+                  - pod
+                  - pod_usage_memory_byte_seconds
+                  - pod_request_cpu_core_seconds
+                  - pod_limit_cpu_core_seconds
+                  - pod_usage_memory_byte_seconds
+                  - pod_request_memory_byte_seconds
+                  - node_capacity_cpu_cores
+                  - node_capacity_cpu_core_seconds
+                  - node_capacity_memory_bytes
+                  - node_capacity_memory_byte_seconds
+                  - pod_limit_memory_byte_seconds
+                  - node_capacity_cpu_cores
+                  type: string
+                maxItems: 11
+                minItems: 1
+                type: array
+              namespace:
+                minLength: 0
+                type: string
+              reportPeriod:
+                description: 'Specifies how to treat concurrent executions of a Job.
```

Comment:
If you wish to make the description multiline and to show values with -, put 
```
description: |
    Specifies how to ... Valid values are:
        - Day
        - ...
```

Reply:
Yes, will take care next time.